lib/src/scss-shared/_nx-input-helpers.scss

```diff
@@ -17,13 +17,25 @@
   }
 
   &:focus-within {
+    .nx-radio__circle {
+      filter: drop-shadow(0 0 3px rgba(2, 166, 217, 0.4));
```

Comment:
When designing the color system don't forget to include this non-opaque color.

Reply:
> When designing the color system don't forget to include this non-opaque color.

@dturner32 ^^